azure/defaults.go

```diff
@@ -100,6 +101,11 @@ const (
 	CustomHeaderPrefix = "infrastructure.cluster.x-k8s.io/custom-header-"
 )
 
+const (
+	// DefaultHTTP429RetryAfter is a default backoff wait time when we get a HTTP 429 response with no Retry-After data.
+	DefaultHTTP429RetryAfter = 1 * time.Minute
```

Comment:
`util/reconcile/defaults.go` is where I was pointing to, sorry should have been explicit!

Reply:
done